apps/labs/posts/building-scipy-with-flang.md

```diff
@@ -252,8 +255,9 @@ Again from that blog post:
 > NumPy stack). The problems for us in the NumPy stack started there and have
 > to be rooted out there as well. This was confirmed for me at the first PyData
 > meetup at Google HQ, where several of us asked Guido [van Rossum; Python's
-> BDFL at the time] what we can do to fix Python packaging for the NumPy stack.
-> Guido's answer was to "solve the problem ourselves".
+>
+> > BDFL at the time] what we can do to fix Python packaging for the NumPy stack.
+> > Guido's answer was to "solve the problem ourselves".
```

Comment:
This seems wrong though - why the extra indentation (switching in the middle of an editor's comment).